tests/test_github_client.py

```
@@ -0,0 +1,417 @@
+    """Test GitHubClient class."""
+
+    @pytest.fixture
+    def mock_github(self):
+        """Create a mock PyGithub instance."""
+        with patch('gh_pr.core.github.Github') as mock_github_class:
+            mock_instance = Mock()
+            mock_github_class.return_value = mock_instance
+            yield mock_instance
+
+    @pytest.fixture
+    def client(self, mock_github):
```

Ruff (0.13.1)

Unused method argument: `mock_github`

(ARG002)